polyhook2/PE/EatHook.hpp

```diff
@@ -16,7 +16,9 @@ namespace PLH {
 class EatHook : public IHook {
 public:
 	EatHook(const std::string& apiName, const std::wstring& moduleName, const char* fnCallback, uint64_t* userOrigVar);
-	EatHook(const std::string& apiName, const std::wstring& moduleName, const uint64_t fnCallback, uint64_t* userOrigVar);
+	EatHook(const std::string& apiName, const std::wstring& moduleName, uint64_t fnCallback, uint64_t* userOrigVar);

```

Comment:
const on the fnCallback argument please

Reply:
But it has no effect on semantics/logic, afaik. A primitive value is always copied, so the argument is always constant from the caller side. It only makes a difference in function body, which is what I have done in the source file.

See: https://stackoverflow.com/a/46292715

Reply:
you are correct, but it's helpful to see the interface as a user that this value is const and it can prevent errors in constructors especially when you may accidentally assign to the parameter. This is more style - as most sprinkling on const is - though there are benefits.